### ISSUE TITLE
[Research] My VA Audit UX Improvements: Create UAT Plan and Conversation Guide

### ISSUE THREAD

**Issue by ericasauv09** [department-of-veterans-affairs/va.gov-team#50648]
## Background
We need to prepare UAT research documents for the My VA Audit initiative. This includes creating the research plan and conversation guide.

## Tasks
- [x] Create UAT research plan
- [x] Create UAT conversation guide

Note: We likely need to create a new UAT folder under the [Audit folder](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/my-va/2022-audit) in GH

## Validation
- [x] Send links to team to let them review asynchronously
- [x] Have another designer review research plan and conversation guide
- [x] Have PM and PO review

**Comment by aagosto90** [comment]
@ajakabcin @Samara-Strauss These are ready for your review! No rush. Thank you! I'll also have another designer review the convo guide; Liz already took a look at the research plan.
[Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/2022-audit/UAT/MyVA-audit-UAT-research-plan.md)
[Conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/2022-audit/UAT/MyVA-audit-UAT-conversation-guide.md)

**Comment by Samara-Strauss** [comment]
Thanks for this! 

The research plan looks great. No notes.

For the conversation guide, remember that the main goal is to make sure the veteran's information is accurate, and that the code is working as expected. While you do ask a question about whether there information is accurate, and there is a task to check links, there are no formal UAT tasks confirming the following:

- Is their military branch of service correct?
- Is their disability rating showing up? Is it correct?
- Are we pulling in their most recent claim?
- Are we showing their next appointment?
- Do they have outstanding debts? Is the information accurate?
- Are we showing their most recent payment? Is it correct?

I know there are a couple usability things to follow up on, and since they are small, that's fine, but remember UAT is about making sure our code works. That is always the top priority for this type of research. This type of research should never be more usability than UAT.

**Comment by aagosto90** [comment]
@Samara-Strauss Thank you so much for reviewing so quickly and sending your feedback! I'm going to take another look at the conversation guide and make some updates so it's focused more around code validation with the usability questions being secondary to that.

I think I have some confusion around what's "in scope" for this project. And when I say in-scope, I realize we don't really care about that but I think it explains why I've left questions out that you've outlined here. I know we audited all of My VA in the beginning of this project, but then for the "audit" updates we only made changes to the following sections: Claims and appeals, health care, education and training, and applications. The nametag hasn't changed as part of this work and we've already completed a separate initiative to make updates to Benefit payments and Outstanding debts and completed all testing to validate those changes. 

I'm of course open to validating the entire page while we have Veterans in these sessions and know that it can't hurt. Just wanted to explain why I haven't put any specific focus on those sections that haven't changed as part of this work.

**Comment by Samara-Strauss** [comment]
> The nametag hasn't changed as part of this work and we've already completed a separate initiative to make updates to Benefit payments and Outstanding debts and completed all testing to validate those changes.

This is a good point, and I should have thought of it! Yes, you can keep questions to just what we have updated.

**Comment by aagosto90** [comment]
@Samara-Strauss 
I've made some tweaks to focus the [conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/2022-audit/UAT/MyVA-audit-UAT-conversation-guide.md) more around validating the code, with the usability questions we're following up on still sprinkled in as secondary tasks that won't take up the majority of the session.

I changed task #2 to "Is the dynamic content on My VA accurate and useful in the sections we've updated for this audit?" which is where I'll focus on validating that Veterans are seeing what they expect to see on My VA and their dynamic content is pulling correctly. I updated that [question](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/2022-audit/UAT/MyVA-audit-UAT-research-plan.md#research-questions) in the research plan too. 

I simplified a bit in the other sections so there will be one or two questions to ask for the education section (if applicable) and applications section, and then we'll spend the majority of the time after that clicking on links to validate analytics. 

Let me know if you have any other feedback; thanks so much!
